# translation of src_debug_server.po to German
# Module src_debug_server translation file for language=de
# Copyright (C) 2003 Archaeopteryx Software, Inc.
# Wing Python IDE -- wingware.com
# Claudia Walenzyk <cwalenzyk@yahoo.com>, 2004.
#
msgid ""
msgstr ""
"Project-Id-Version: src_debug_server\n"
"POT-Creation-Date: 2018-03-21 10:16+EDT\n"
"PO-Revision-Date: 2016-12-19 20:56+0100\n"
"Last-Translator: Christoph Heitkamp <info@chrisheitkamp.de>\n"
"Language-Team: German\n"
"Language: de_DE\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"Generated-By: pygettext.py 1.4\n"
"X-Generator: Poedit 1.8.11\n"

#: src/debug/tserver/abstract.py:331
msgid "Wing Debug Log File\n"
msgstr "Wing Debug-Protokolldatei\n"

#: src/debug/tserver/abstract.py:332
msgid "This file can be used to diagnose problems in debugger internals\n"
msgstr ""
"Diese Datei kann genutzt werden, um Probleme in der Debugger-"
"Systemarchitektur festzustellen.\n"

#: src/debug/tserver/abstract.py:333
msgid "For help, contact support@wingware.com\n"
msgstr ""
"Wenn Sie Hilfe benötigen, kontaktieren Sie uns bitte über support@wingware."
"com.\n"

#: src/debug/tserver/abstract.py:338
msgid "Warning:  Log file does not exist:\n"
msgstr "Warnung: Protokolldatei existiert nicht:\n"

#: src/debug/tserver/abstract.py:340
msgid "Warning:  and could not be created; logging disabled\n"
msgstr ""
"Warnung: und konnte nicht erstellt werden; Protokollierung deaktiviert\n"

#: src/debug/tserver/abstract.py:343
msgid "Warning:  Log file is not a file:\n"
msgstr "Warnung: Protokolldatei ist keine Datei:\n"

#: src/debug/tserver/abstract.py:345
msgid "Warning:  Logging disabled\n"
msgstr "Warnung: Protokollierung deaktiviert\n"

#: src/debug/tserver/dbginspect.py:54 src/debug/tserver/dbgserver.py:2906
#: src/debug/tserver/dbgutils.py:443 src/debug/tserver/dbgutils.py:463
msgid "(truncated)"
msgstr ""

#: src/debug/tserver/dbginspect.py:920 src/debug/tserver/dbgutils.py:451
#: src/debug/tserver/dbgutils.py:473 src/debug/tserver/dbgutils.py:2949
msgid "<opaque "
msgstr "<unlesbar "

#: src/debug/tserver/dbgserver.py:1176
msgid "<module>"
msgstr "<Modul>"

#: src/debug/tserver/dbgserver.py:1324
msgid "<error evaluating>"
msgstr "<Fehler beim Bewerten>"

#: src/debug/tserver/dbgserver.py:1897 src/debug/tserver/dbgserver.py:1935
msgid "<thread %i is running>"
msgstr "<Thread %i läuft>"

#: src/debug/tserver/dbgserver.py:1900 src/debug/tserver/dbgserver.py:1938
msgid "<invalid thread id %s or stack index %s>"
msgstr "<ungültige Thread-id %s oder Stack-Index %s>"

#: src/debug/tserver/dbgserver.py:1902 src/debug/tserver/dbgserver.py:1940
msgid "<invalid stack frame>"
msgstr "<ungültiger Stack-Frame>"

#: src/debug/tserver/dbgserver.py:1968 src/debug/tserver/dbgserver.py:1997
msgid "<sys.modules does not exist>"
msgstr "<sys.modules existiert nicht>"

#: src/debug/tserver/dbgserver.py:1971 src/debug/tserver/dbgserver.py:2003
msgid "<error accessing sys.modules>"
msgstr "<Fehler beim Zugreifen auf sys.modules>"

#: src/debug/tserver/dbgserver.py:2000
msgid "<no longer exists"
msgstr "<existiert nicht mehr"

#: src/debug/tserver/dbgserver.py:2048 src/debug/tserver/dbgserver.py:2101
msgid "<no longer exists>"
msgstr "<existiert nicht mehr>"

#: src/debug/tserver/dbgserver.py:2079 src/debug/tserver/dbgserver.py:4106
msgid "<parent too large>"
msgstr "<parent zu lang>"

#: src/debug/tserver/dbgserver.py:2081 src/debug/tserver/dbgserver.py:4108
msgid "<invalid parent type>"
msgstr "<ungültiger übergeordneter Datentyp>"

#: src/debug/tserver/dbgserver.py:2091 src/debug/tserver/dbgserver.py:2543
msgid "<value not found>"
msgstr "<Wert nicht gefunden>"

#: src/debug/tserver/dbgserver.py:2531
msgid "<thread is running>"
msgstr "<Thread läuft>"

#: src/debug/tserver/dbgserver.py:2533
msgid "<invalid context>"
msgstr "<ungültiger Kontext>"

#: src/debug/tserver/dbgserver.py:4114
msgid "<invalid path %s at pos=%i>"
msgstr "<ungültiger Pfad %s an pos=%i>"

#: src/debug/tserver/dbgutils.py:150
msgid "<undefined>"
msgstr ""

#: src/debug/tserver/dbgutils.py:166
msgid "at"
msgstr ""

#: src/debug/tserver/dbgutils.py:174
msgid "dim=%sx%s>"
msgstr ""

#: src/debug/tserver/dbgutils.py:176
msgid "len=%s>"
msgstr ""

#: src/debug/tserver/dbgutils.py:186 src/debug/tserver/dbgutils.py:2451
#, fuzzy
msgid "<error handling value>"
msgstr "<Fehler beim Bewerten>"

#: src/debug/tserver/dbgutils.py:188
msgid "<timeout>"
msgstr ""

#: src/debug/tserver/dbgutils.py:196
msgid "<huge "
msgstr ""

#: src/debug/tserver/dbgutils.py:216
msgid "<long "
msgstr ""

#: src/debug/tserver/dbgutils.py:233
#, fuzzy
msgid "<opaque"
msgstr "<unlesbar "

#: src/debug/tserver/dbgutils.py:465
msgid "<huge %s at %s; len=%i>"
msgstr "<huge %s an %s; Länge=%i>"

#: src/debug/tserver/dbgutils.py:653 src/debug/tserver/dbgutils.py:665
msgid ", pos "
msgstr ", pos "

#: src/debug/tserver/dbgutils.py:669
msgid ", line "
msgstr ", Zeile "

#: src/debug/tserver/dbgutils.py:671
msgid "SyntaxError: "
msgstr "SyntaxFehler:"

#: src/debug/tserver/dbgutils.py:699
msgid "Error"
msgstr "Fehler"

#: src/debug/tserver/dbgutils.py:707
msgid "Error: Wing was unable to pack up an exception message"
msgstr "Fehler: Wing konnte Exception-Nachricht nicht zusammenpacken"

#: src/debug/tserver/dbgutils.py:3258 src/debug/tserver/dbgutils.py:3274
msgid "Key"
msgstr ""

#: src/debug/tserver/dbgutils.py:3258 src/debug/tserver/dbgutils.py:3274
msgid "Value"
msgstr ""

#: src/debug/tserver/dbgutils.py:3269 src/debug/tserver/dbgutils.py:3296
msgid "Item"
msgstr ""

#: src/debug/tserver/netserver.py:459
msgid "Redirected I/O failed on closed debug connection"
msgstr "Umgeleiteter I/O an geschlossener Debug-Verbindung gescheitert"

#: src/debug/tserver/netserver.py:613
msgid "EOF when reading a line"
msgstr "Eingabe eines Dateiende-Zeichens (EOF) beim Lesen einer Zeile"

#: src/debug/tserver/netserver.py:1110
msgid "(unavailable)"
msgstr "(nicht verfügbar)"

#: src/debug/tserver/netserver.py:1366
msgid "[omitting some output]"
msgstr "[Ausgabe wird teilweise ausgelassen]"

#: src/debug/tserver/startdebug.py:195 src/debug/tserver/startdebug.py:373
#, fuzzy
msgid "wingdb.py: Error: Debug file does not exist:"
msgstr "Warnung: Protokolldatei existiert nicht:\n"

#: src/debug/tserver/startdebug.py:199 src/debug/tserver/startdebug.py:377
#, fuzzy
msgid "wingdb.py: Error: Debug file is not a file:"
msgstr "Warnung: Protokolldatei ist keine Datei:\n"

#: src/debug/tserver/startdebug.py:426
msgid "Network peer is "
msgstr ""

#: src/debug/tserver/startdebug.py:427
msgid "Attach port = "
msgstr ""

#: src/debug/tserver/startdebug.py:428
msgid "Initial stop = "
msgstr ""

#: src/debug/tserver/startdebug.py:442
msgid "wingdb.py: Could not create debug server"
msgstr ""

#: src/debug/tserver/startdebug.py:464
msgid "wingdb.py: Running %s"
msgstr ""

#: src/debug/tserver/startdebug.py:475
msgid "wingdb.py: Server exiting abnormally on exception"
msgstr ""

#: src/debug/tserver/startdebug.py:635
msgid "wingdb.py: Error: Failed to start the debug server"
msgstr ""

#: src/debug/tserver/startdebug.py:636
msgid "wingdb.py: Error: You may be running an unsupported version of Python"
msgstr ""

#: src/debug/tserver/startdebug.py:637
msgid "wingdb.py: Python version = %s"
msgstr ""

#: src/debug/tserver/startdebug.py:717 src/debug/tserver/startdebug.py:719
msgid "-- Type return or enter to exit --\n"
msgstr ""

#~ msgid "<unknown type>"
#~ msgstr "<unbekannter Typ>"

#~ msgid "Value too large to evaluate: "
#~ msgstr "Wert zu groß für Bewertung: "

#~ msgid "<invalid path registration>"
#~ msgstr "<ungültige Pfadregistrierung>"

#~ msgid "Function"
#~ msgstr "Funktion"

#~ msgid "Method"
#~ msgstr "Methode"

#~ msgid "Could not execute because an error occurred:\n"
#~ msgstr ""
#~ "Programm konnte nicht ausgeführt werden, weil ein Fehler aufgetreten "
#~ "ist:\n"

#~ msgid "Error: Runtime exception occurred"
#~ msgstr "Fehler: Laufzeit-Exception aufgetreten"
